### ISSUE TITLE
mercury in jupyterlab

### ISSUE THREAD

**Issue by stametsm** [mljar/mercury#218]
Unsure if I am doing this right, but I am trying to run notebooks from jupyterlab using mercury and it is not working.

This is the example I get while trying to run the demo(same problem occurs when trying to run other notebooks as well)
```
The notebook demo-dataframe-and-plots.ipynb will be updated
Successfully updated a notebook (id:18)
The notebook demo-slides.ipynb will be updated
Successfully updated a notebook (id:19)
The notebook demo.ipynb will be updated
Successfully updated a notebook (id:20)
Traceback (most recent call last):
  File "C:\Users\matth\AppData\Roaming\jupyterlab-desktop\jlab_server\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\matth\AppData\Roaming\jupyterlab-desktop\jlab_server\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\matth\AppData\Roaming\jupyterlab-desktop\jlab_server\Scripts\mercury.exe\__main__.py", line 7, in <module>   
  File "C:\Users\matth\AppData\Roaming\jupyterlab-desktop\jlab_server\lib\site-packages\mercury\mercury.py", line 176, in main
    worker = subprocess.Popen(
  File "C:\Users\matth\AppData\Roaming\jupyterlab-desktop\jlab_server\lib\subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\matth\AppData\Roaming\jupyterlab-desktop\jlab_server\lib\subprocess.py", line 1311, in _execute_child        
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

I tried to find what was causing the issue(mercury or where everything is installed) but here is the place that it is installed using pip:
```
Location: c:\users\matth\appdata\roaming\jupyterlab-desktop\jlab_server\lib\site-packages
```

If anyone could tell me whats happening between mercury and jupyterlab that would be awesome(also other pip install stuff is fine)

**Comment by stametsm** [comment]
Hello!

Somehow I was able to get it to work, but wasn't exactly sure how.

I had installed mercury through the console, and neither cmd or the terminal file in jupyterlab was working to run mercury. I am not sure what I did, but I have now been able to launch mercury through cmd specifying the notebook also used in jupyterlab(though this originally wasn't the case when I first listed my problem). It still does not working when trying to turn though terminal in jupyterlab, giving the same bug listed above.
